src/test/java/io/jenkins/plugins/intotorecorder/transport/TransportTest.java

```diff
@@ -41,5 +41,10 @@ public void testTransportFactory() throws Exception {
         someTransport = Transport.TransportFactory.transportForURI(testUrl);
         assertTrue(someTransport instanceof GenericCRUD);
 
+        /* test creating Grafeas transports */
+        testUrl = new URI("grafeas+http://localhost:8080/v1beta1/projects/provider_example/occurrences?noteName=noteName&resourceUri=resourceUri");
+        someTransport = Transport.TransportFactory.transportForURI(testUrl);
+        assertTrue(someTransport instanceof Grafeas);
+
```

Comment:
cool test! although we may want to at some point in the future test with some more uri's...

Reply:
https://github.com/jenkinsci/in-toto-plugin/issues/12

Collected here for now.